cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -42,29 +41,56 @@ class Deploy(BaseSalesforceMetadataApiTask):
         },
     }
 
+    def _get_api(self, path=None):
+        if not path:
+            path = self.task_config.options__path
+
+        package_zip = self._get_package_zip(path)
+        self.logger.info("Payload size: {} bytes".format(len(package_zip)))
+        return self.api_class(self, package_zip, purge_on_delete=False)
+
+    def _include_directory(self, root_parts):
+        # include the root directory, all non-lwc directories and sub-directories, and lwc component directories
+        return len(root_parts) == 0 or root_parts[0] != "lwc" or len(root_parts) == 2
+
+    def _include_file(self, root_parts, f):
+        if len(root_parts) == 2 and root_parts[0] == "lwc":
+            # is file of lwc component directory
+            lower_f = f.lower()
+            return (
+                lower_f.endswith(".js")
+                or lower_f.endswith(".js-meta.xml")
+                or lower_f.endswith(".html")
+                or lower_f.endswith(".css")
+                or lower_f.endswith(".svg")
+            )
+        return True
+
+    def _get_files_to_package(self, path):
```

Comment:
Originally, the logic inside of `_get_files_to_package` was contained in `_get_package_zip`, but the asserts in the test give anything helpful if they failed since `_get_files_to_package` returns a base 64 encoded string.   I created `_get_files_to_package` so I could easily assert what files were included in the relative path.